site/_guides/prometheus.md

```diff
@@ -21,8 +21,11 @@ port `8002`.
 
 ## Contour Metrics
 
-Contour exposes a Prometheus-compatible `/metrics` endpoint on port `8000` with
-the following metrics:
+Contour exposes a Prometheus-compatible `/metrics` endpoint defaulted to `8000` this is configurable using the `--http-address` and `--http-port` flags for the `serve` command.
```

Comment:
"that defaults to listening on port `8000`. This can be configured by using ..."